### DISCUSSION TITLE
O que fazer se a empresa que vc trabalhou remover seus vídeos do YouTube?

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#1284] [Discussão]
Oi pessoal,

> Por causa de uma série de eventos inesperados que causaram uma instabilidade no sistema

3 vídeos que publiquei no canal de uma empresa privada foram removidos, o que mais me doeu é que fizemos os vídeos para fomentar a comunidade e foi removido de forma unilateral sem me consultar, nem comunicar da remoção, e só fiquei sabendo do ocorrido depois de receber elogios ao meu conteúdo durante uma conferência de JavaScript.

A questão é: como devo prosseguir? Devo insistir que coloquem o conteúdo novamente? Devo colocar os vídeos em outro canal? Vocês já passaram por isso?

Abs.

**Comment by luizbills** [comment]
Tem algum contrato que possa te ajudar? Se não tiver, vai depender da boa vontade da empresa (o que pode signficar: "deixa para lá").

**Comment by iagocavalcante** [comment]
Cara, publica no seu canal e continue fomentando a comunidade, quem perde é só a empresa ^^

**Comment by felipefialho** [comment]
Muitas empresas colocam no contrato que todo o conteúdo produzido por você é de propriedade intelectual dela. Claro que por questões de bom senso, essas coisas não são levadas ao pé da letra pela maioria delas.

Mas como o caso foi de remoção, não apropriação, tenho algumas perguntas:

Você tem o vídeo salvo? Eles tem o backup disso?

Se sim, vale conversar para subir novamente na sua conta pessoal.

**Comment by evaporei** [comment]
@felquis , já chegou a tentar falar com a empresa do motivo deles terem sido removidos? Caso eles realmente não venham voltar esses vídeos no canal, talvez você possa ao menos conseguir os arquivos com eles.
